.github/workflows/test-docs.yml

```
@@ -0,0 +1,33 @@
+    branches:
+      - "main"
+jobs:
+  test-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "20.11.1"
+      - name: Cache Yarn dependencies
+        uses: actions/cache@v2
```

actionlint

the runner of "actions/cache@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)